tools/scripts/publish-dist-set-vars.test.mjs

```diff
@@ -0,0 +1,170 @@
+import {
+  validateAppName,
+  AppsThatDeployToMainnetFromDevelop,
+  S3BucketNameForApp,
+  getBucketName,
+  getEnvironmentFromDevelop,
+} from './publish-dist-set-vars.mjs';
+import { test } from 'tape';
```

Comment:
Was there a particular reason for using tape? I think it would be beneficial to stick with jest as thats what we've been using across the project already

Reply:
I didn't like doing this (Jest takes seconds to start up, Tape is nice and quick and doesn't rely on globals and all sorts of things) but it has the side benefit of making it easy to add in to the main test suite. 

Done 👍